### PR TITLE
CI: Recursively clear node modules when building for ci

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Description
Following developit's advice from here: https://github.com/preactjs/compressed-size-action/issues/59#issuecomment-865131410

## How has this been tested?
Compressed size check action should continue to pass.

## Types of changes
Bug fix/adjust compressed size check to work with lerna.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [N/A] My code follows the accessibility standards. 
- [N/A] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: .github/workflows/bundle-size.yml [hunk 1]
```diff
@@ -55,7 +55,8 @@ jobs:
                   path: ~/.npm
                   key: ${{ runner.os }}-node-${{ matrix.node }}-npm-cache-${{ hashFiles('**/package-lock.json') }}
 
-            - uses: preactjs/compressed-size-action@7d87f60a6b0c7d193b8183ce859ed00b356ea92f # v2.1.0
+            - uses: preactjs/compressed-size-action@df6e03e187079aef959a2878311639c77b95ee2e # v2.2.0
               with:
                   repo-token: '${{ secrets.GITHUB_TOKEN }}'
                   pattern: '{build/**/*.min.js,build/**/*.css}'
+                  clean-script: 'distclean'
```
Comment (ciampo) [inline .github/workflows/bundle-size.yml:62]: Will the `distclean` remove ALL of the `node_modules/` folders that we need to clear in this case?

**Review by ciampo — APPROVED** [review]
Changes LGTM and follow the instructions from https://github.com/preactjs/compressed-size-action/pull/60 🚀

File: .github/workflows/bundle-size.yml (hunk 1 above)
Comment (sarayourfriend) [reply .github/workflows/bundle-size.yml:62]: Yes, it runs `rimraf packages/*/node_modules` 🙂

Comment (ciampo) [reply .github/workflows/bundle-size.yml:62]: Nice!

**Pull request merged**